packages/platform-android/src/commands/runAndroid/index.js

```diff
@@ -359,7 +359,7 @@ export default {
       command: '--terminal [string]',
       description:
         'Launches the Metro Bundler in a new window using the specified terminal path.',
-      default: '',
+      default: undefined,
```

Comment:
Let's make it `default: process.env.REACT_TERMINAL || process.env.TERM_PROGRAM` and remove the default assignement on line 231.
```suggestion
      default: process.env.REACT_TERMINAL || process.env.TERM_PROGRAM,
```